### DISCUSSION TITLE
Unable to reach Microsoft OneDrive API service at this point in time, re-trying network tests

### DISCUSSION THREAD

**Discussion by andmunoz** [abraunegg/onedrive#2669] [General]
Today I'm receiving this error message when trying to sync with --monitor option:

```
Unable to reach Microsoft OneDrive API service at this point in time, re-trying network tests

Cannot connect to Microsoft OneDrive Service - Network Connection Issue

ERROR: Microsoft OneDrive API returned an error with the following message:
  Error Message:    Timeout was reached on handle 5C962A0C3930
  Calling Function: testNetwork()
```

Do you have the same problem?

Thanks for your advice.

**Comment by abraunegg** [comment]
@andmunoz 
This is indicative of a network access issue between you are Microsoft OneDrive API service.

Please read: https://github.com/abraunegg/onedrive/wiki/Frequently-Asked-Questions#cannot-connect-to-microsoft-onedrive-service---network-connection-issue

### Actions for you:
1. Ensure your Linux distribution is as current as possible - do not be using obsolete Linux distributions
2. Ensure you are running the most current available 'curl' version possible for your platform. 
3. Ensure you are using the correct client version - v2.4.25 or above
4. Validate your DNS and HTTPS access to the required services (see above URL)

**Reply by andmunoz** [reply]
Thanks for yout answer, but i do not anything (on friday it seems is working) only happens i must have to restart my computer. I run a testing and have a timeout: 

```
curl: (28) Failed to connect to login.microsoftonline.com port 443 after 4983 ms: Expiró el tiempo de conexión
curl: (28) Failed to connect to graph.microsoft.com port 443 after 4426 ms: Expiró el tiempo de conexión
```

It's strange because i connect to that computer via ssh but it seems had a network issue. I will review that for now.

Thanks abraunegg

**Reply by abraunegg** [reply]
@andmunoz 
It appears that you have a firewall blocking your access.

**Reply by andmunoz** [reply]
Yes, i guess... I hope recover my computer soon :-(
